setup/monitoring/index.md

```diff
@@ -6,3 +6,262 @@ sidebar:
 ---
 
 {% include toc %}
+Spinnaker is instrumented with numerous metrics internally so that it can be
+monitored. Monitoring spinnaker typically involves the spinnaker-monitoring
+daemon, awhich collects metrics reported by each microservice instance and
+reports them to a third-party monitoring system which you then use to view
+overview dashboards, receive alerts, and informally browse depending on your
+needs.
+
+Spinnaker publishes internal metrics using a multi-dimensional data model
+based on "tags". The metrics, data-model, and usage a discussed further
+in the sections [Consuming Metrics](#consuming-metrics) and in the
+[Monitoring Reference document](/reference/monitoring/).
+
+Spinnaker currently supports three specific third-party systems: 
+[Datadog](https://www.datadoghq.com/), [Prometheus](https://prometheus.io/)
+and [Stackdriver](http://www.stackdriver.com/). The daemon is extensible so
+that it should be straight forward to add other systems as well. In fact,
+each of the supported systems was provided using the daemon's extension
+mechanisms -- there are not "native" systems.
+
+You can also use the microservice HTTP endpoint `/spectator/metrics`
+directly to scrape metrics yourself. The JSON document structure is 
+further documented in the [Monitoring reference section](/reference/monitoring/).
+
+
+## Installing Spinnaker Monitoring (on VMs)
```

Comment:
Let's talk tomorrow in person about how to split divide this work w.r.t. work done by halyard vs work done by the user.